backends/postgres/postgres_backend_test.go

```diff
@@ -410,6 +410,71 @@ func TestCron(t *testing.T) {
 	})
 }
 
+func TestMultipleCronNodes(t *testing.T) {
+	jobsProcessed := sync.Map{}
```

Comment:
I ended up switching to this over a channel so that I can detect duplicate job consumption by the job ID. Since we know that only one job can be inserted at a time for a given cron, we need to make sure that that one job is only consumed once.

The test just checks to see that when the job is processed, it hasn't already been consumed by someone else.